.github/workflows/create-release-prs.yml

```diff
@@ -42,21 +42,26 @@ jobs:
         run: invoke lint
       - name: Run format check
         run: invoke format
+        
+      - name: Configure git
+        run: git config --global user.email "tamr-toolbox@tamr.com" && git config --global user.name "Tamr"
+      - name: Init release branch
+        run: git checkout -b 'release-${{ steps.get_version.outputs.version }}'
+        
+      - name: Remove previous latest doc
+        run: rm -r docs/latest
+      - name: Delete previous latests docs
+        run: git add . && git commit -m 'remove previous latest docs'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build docs
         run: invoke docs
-
-      - name: Remove previous latest doc
-        run: rm -r docs/latest
+          
       - name: Move docs to online pages location
         run: cp -r docs/_draft_build docs/${{ steps.get_version_short.outputs.version }} && cp -r docs/_draft_build docs/latest
 
-      - name: Configure git
-        run: git config --global user.email "tamr-toolbox@tamr.com" && git config --global user.name "Tamr"
-      - name: Init release branch
-        run: git checkout -b 'release-${{ steps.get_version.outputs.version }}'
-
-      - name: Commit changes
```

Comment:
It looks like you are moving up the "Configure git" step and adding an automatic commit that removes the previous latest docs, before latest docs are built.  Does that entirely describe the change here?

Reply:
@skalish That is correct. I believe adding an explicit delete commit will resolve the issue. In order to add this, I needed to relocate the commands establishing git and the delete itself